src/components/accordion/__styles__/index.ts

```diff
@@ -12,15 +12,14 @@ export const ElAccordionItem = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  cursor: pointer;
   padding: 0.875rem 0;
   border-bottom: 1px solid var(--color-grey-100);
 
   &:has(+ .${elIsActive}) {
     border-bottom: none;
   }
 `
-export const ElAccordionTitle = styled.div`
+export const ElAccordionTitle = styled.a`
```

Comment:
The anchor should be on the accordion item, not on the title - the item is the clickable, not the title